src/components/StatusDashboard/current_migrations.jsx

```diff
@@ -192,12 +192,18 @@ function TableContent({ collapsed, name, resort, rows, select, sort, fetched })
           return (
             <tr key={row.name}>
               <td>
-                <Link href={href}
-                  style={{ display: "block" }}
-                  onClick={event => {
-                    event.preventDefault();
-                    setState(href);
-                  }}>{row.name}</Link>
+                {row.success ?
+                  <Link href={href}
+                    style={{ display: "block" }}
+                    onClick={event => {
+                      event.preventDefault();
+                      setState(href);
+                    }}>{row.name}</Link>
+                : <>
+                    <span title="Failed to load. Check console." style={{cursor: "pointer"}}>⚠️</span>
```

Comment:
```suggestion
                    <span title="Failed to load. Refresh the page to try again." style={{cursor: "pointer"}}>⚠️</span>
```

I don't think the "check console" adds much and many folks won't understand what that means. We could direct them to refresh the page maybe?